tests/query_test.go

```diff
@@ -40,3 +89,124 @@ func (s *QuerySuite) TestQuery() {
 		s.Equal("bar", store.MustFindOne(queryErr).Foo)
 	})
 }
+
+func (s *QuerySuite) TestFindById() {
+	store := NewQueryFixtureStore(s.db)
+
+	docName := "bar"
+	doc := NewQueryFixture(docName)
+	s.Nil(store.Insert(doc))
+
+	query := NewQueryFixtureQuery()
+	query.FindByID(doc.ID)
+	s.NotPanics(func() {
+		s.Equal(docName, store.MustFindOne(query).Foo)
+	})
+
+	queryManyId := NewQueryFixtureQuery()
+	queryManyId.FindByID(queryFixtures[1].ID, queryFixtures[2].ID)
+	count, err := store.Count(queryManyId)
+	s.Equal(2, int(count))
+	s.Nil(err)
+
+	notID := kallax.NewID()
+	queryErr := NewQueryFixtureQuery()
+	queryErr.FindByID(notID)
+	s.Panics(func() {
+		store.MustFindOne(queryErr)
+	})
+}
+
+func (s *QuerySuite) TestFindBy() {
+	store := NewQueryFixtureStore(s.db)
+	s.NotPanics(func() {
+		//s.True(store.MustFindOne(NewQueryFixtureQuery().FindByStringProperty("StringProperty1")).Eq(queryFixtures[1]))
+	})
+	s.Panics(func() {
+		store.MustFindOne(NewQueryFixtureQuery().FindByStringProperty("NOT_FOUND"))
+	})
+
+	s.NotPanics(func() {
+		//s.True(store.MustFindOne(NewQueryFixtureQuery().FindByBoolean(false)).Eq(queryFixtures[1]))
```

Comment:
nope,
Those tests are already broken tests, affected by https://github.com/src-d/go-kallax/issues/96

I updated it in my rebased branch, and it looks this way:
![image](https://cloud.githubusercontent.com/assets/2437584/23071509/da4248de-f52e-11e6-85d3-a3db2e928d13.png)